include/fmt/std.h

```diff
@@ -228,6 +232,33 @@ struct formatter<std::optional<T>, Char,
 FMT_END_NAMESPACE
 #endif  // __cpp_lib_optional
 
+#ifdef __cpp_lib_source_location
+FMT_BEGIN_NAMESPACE
+FMT_EXPORT
+template <typename Char>
+struct formatter<std::source_location, Char,
```

Comment:
`std::source_location` can only be formattable with `char` so let's remove the `Char` template parameter.